src/components/KCollapse/KCollapse.vue

```diff
@@ -0,0 +1,243 @@
+<template>
+  <div class="k-collapse w-100">
+    <div
+      :class="{
+        'd-flex': trailingTrigger,
+        'd-block': !trailingTrigger
+      }"
+      class="k-collapse-heading w-100 mb-3"
```

Comment:
nit: I don't believe you need the `width: 100%` here.
```suggestion
      class="k-collapse-heading mb-3"
```

Reply:
Fixed